mq/main/pom.xml

```diff
@@ -908,13 +908,13 @@
 
 	    <!-- compile test only -->
             <dependency>
-                <groupId>javax.activation</groupId>
+                <groupId>jakarta.activation</groupId>
                 <artifactId>activation</artifactId>
```

Comment:
Seems non-existent in current `${jaf.version}`.
1. Looks like unused managed-only dependency, that could be removed anyway.
1. On the other hand - JAF will be needed for migration to JDK11 in `packager-opensource`, so perhaps artifactId should be updated to `jakarta.activation-api` then?

But I would remove it now.